mdgeom/__init__.py

```diff
@@ -6,4 +6,4 @@
 # Add imports here
 from importlib.metadata import version
 
-__version__ = version("mdgeom")
+__version__ = version("mdgeomkit")
```

Comment:
This is really messy - if you call the package `mdgeomkit` then it'll expect `mdgeomkit` here not `mdgeom`, etc...

Reply:
That's something the cookie-cutter needs to get right.

Reply:
The cookie cutter has three names:
- project_name, which can be anything including with spaces, e.g. "An MDAnalysis toolkit for geometries"
- repo_name, which is used for the GitHub URL so cannot include spaces but can include, e.g. hyphens, so `md-geom-kit` is allowed
- package_name, which must be Python package name compatible so no hyphens allowed. This should be used for `import mdgeomkit` and the like.

It seems to me that the `pip install xxx` name should either be the repo_name or package_name for simplicity -- is there much use for an additional name option here?

Reply:
I thought that `project_name` was the PyPi package name, `repo_name` would usually equal `project_name` and I understand `package_name` to be exactly what you describe.

I agree with you (I think) that we could just have `package_name == repo_name` and only have `package_name`.